sysutils/monit/src/opnsense/mvc/app/models/OPNsense/Monit/Monit.xml

```diff
@@ -46,6 +46,57 @@
             <default>0</default>
             <Required>Y</Required>
          </ssl>
+         <logfile type="TextField">
+            <Required>N</Required>
+            <default>syslog facility log_daemon</default>
+            <mask>/^(\/[^\/ ]*)+\/?|syslog.*$/</mask>
+            <ValidationMessage>Should be a valid absolute path to the log file or the keyword "syslog".</ValidationMessage>
+         </logfile>
+         <statefile type="TextField">
+            <Required>N</Required>
+            <mask>/^(\/[^\/ ]*)+\/?$/</mask>
+            <ValidationMessage>Should be a valid absolute path to the state file.</ValidationMessage>
+         </statefile>
+         <eventqueuePath type="TextField">
+            <Required>N</Required>
+            <mask>/^(\/[^\/ ]*)+\/?$/</mask>
+            <ValidationMessage>Should be a valid absolute path to the eventqueue directory.</ValidationMessage>
+         </eventqueuePath>
+         <eventqueueSlots type="IntegerField">
+            <Required>N</Required>
+            <ValidationMessage>Eventqueue Slots must be a numeric value.</ValidationMessage>
+         </eventqueueSlots>
+         <httpdEnabled type="BooleanField">
+            <default>0</default>
+            <Required>Y</Required>
+         </httpdEnabled>
+         <httpdUsername type="TextField"/>
+         <httpdPassword type="TextField"/>
+         <httpdPort type="IntegerField">
+            <default>2812</default>
+            <Required>Y</Required>
+            <MinimumValue>0</MinimumValue>
+            <MaximumValue>65535</MaximumValue>
+            <ValidationMessage>Local Monit Port needs to be an integer value between 0 and 65535</ValidationMessage>
```

Comment:
0 probably automatic? otherwise valid ports are starting at 1

Reply:
Right, port 0 is reserved.